src/main/java/spireMapOverhaul/zones/monsterZoo/BackgroundMonsterEffect.java

```diff
@@ -0,0 +1,373 @@
+package spireMapOverhaul.zones.monsterZoo;
+
+import basemod.ReflectionHacks;
+import basemod.abstracts.CustomMonster;
+import basemod.animations.AbstractAnimation;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.MathUtils;
+import com.badlogic.gdx.math.Vector2;
+import com.esotericsoftware.spine.Bone;
+import com.esotericsoftware.spine.Skeleton;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.monsters.beyond.*;
+import com.megacrit.cardcrawl.monsters.city.*;
+import com.megacrit.cardcrawl.monsters.exordium.*;
+import com.megacrit.cardcrawl.vfx.AbstractGameEffect;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+//Mostly copied from https://github.com/a-personal-account/PaleOfTheAncients/blob/master/src/main/java/paleoftheancients/finarubossu/vfx/BackgroundMonster.java
+public class BackgroundMonsterEffect extends AbstractGameEffect {
+    private AbstractCreature ac;
+    private Skeleton skeleton;
+    private Texture img;
+    private AbstractAnimation animation;
+    private Vector2 direction;
+
+    public BackgroundMonsterEffect(boolean isfast) {
+        this.ac = RandomEntity();
+        this.ac.flipHorizontal = MathUtils.randomBoolean();
+
+
+        this.direction = new Vector2(MathUtils.random(100, 200), 0);
+        this.direction = direction.nor();
+        this.renderBehind = true;
+        this.color = Color.WHITE.cpy().sub(0, 0, 0, MathUtils.random(0.4F, 0.7F));
+        if (MathUtils.randomBoolean(0.2F)) {
+            this.direction.x *= -1;
+            this.color.a -= 0.1F;
+        }
+        this.direction.x *= Settings.scale * 50F;
+        this.direction.y *= Settings.scale * 50F;
+
+        float multiplier = Settings.WIDTH / 2F / Math.abs(direction.x) * 1.3F;
+        this.ac.drawX = Settings.WIDTH / 2F - direction.x * multiplier;
+        this.ac.drawY =  MathUtils.random(Settings.HEIGHT * 0.25f, Settings.HEIGHT * 0.75f);
+
+        this.rotation = 0;
```

Comment:
I see that the rotation velocity logic below is commented out, which I assume is to avoid the potential distraction of having enemy images spinning in the background as they go by.

I checked this out locally and while not having active rotation makes sense, it might be cool if the enemies had some rotation applied. If you don't want enemies to be upside down, you could just restrict the range of rotation so that the enemies are always still upright, just tilted.